### ISSUE TITLE
serialise depends on binded this (missing pluralize dependency)

### ISSUE THREAD

**Issue by wopian** [wopian/kitsu#129]
Unable to serialise data using `kitsu-core` as a standalone - tests did not pick this up as they're pre-bound in the specs for each bind combination.

Should fallback to requiring `pluralize` if `this.plural` is not a function

Ref: https://codepen.io/wopian/pen/KQZwzq (console)

**Comment by wopian** [comment]
Work around until a future release

## With pluralization enabled

```shell
yarn add pluralize
```

```js
import { serialise, camel, kebab } from 'kitsu-core'
import plural from 'pluralize'

const output = await serialise.apply({ camel, resCase: kebab, plural }, [ model, obj, 'PATCH' ])
```

## Without pluralization

```js
import { serialise, camel, kebab } from 'kitsu-core'

const output = await serialise.apply({ camel, resCase: kebab, plural: s => s }, [ model, obj, 'PATCH' ])
```

**Comment by wopian** [comment]
Fixed in 9.0.0
